scanpipe/pipelines/inspect_elf_binaries.py

```diff
@@ -27,7 +27,7 @@
 from scanpipe.pipelines import Pipeline
 
 
-class InspectElfBinaries(Pipeline):
+class InspectELFBinaries(Pipeline):
```

Comment:
You need to update the references to this as well.

Reply:
Done!